pkg/osx/scripts/postinstall

```diff
@@ -34,10 +34,20 @@ if [ ! -f /etc/salt/minion ]; then
 fi
 
 ###############################################################################
+# Create symlink to salt-config.sh
+###############################################################################
+# echo "Symlink: Creating symlink for salt-config..." >> /tmp/postinstall.txt
+if [ ! -d "/usr/local/sbin" ]; then
+    mkdir /usr/local/sbin
```

Comment:
We really want to create `/usr/local/sbin` on a system just to store one config script?